lib/rubocop/cop/style/redundant_argument.rb

```diff
@@ -40,6 +40,7 @@ class RedundantArgument < Cop
         MSG = 'Argument %<arg>s is redundant because it is implied by default.'
 
         def on_send(node)
+          return if node.arguments.count != 1
```

Comment:
I think you should also return directly if there's no explicit receiver. I don't see a check for this currently.

Reply:
added